.github/workflows/deploy_production_appstore.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
       with:
-        fetch-depth: 0
+        submodules: recursive
```

Comment:
Why you changed to `recursive`?

Reply:
@suho It supposes to fetch our submodules, such as: `jsonapi-kotlin`

Reply:
@suho reverted it 🙏